docs/src/components/NavigationFooter/NavigationFooter.scss

```diff
@@ -26,6 +26,11 @@
   color: $bpk-color-white;
   text-decoration: none;
 
+  @include bpk-breakpoint-mobile {
```

Comment:
Why are we only doing this for mobile? Surely it should be vertically centred on desktop too?

Reply:
Not according to the designs